Veeam/PRTG-VeeamBRStats.ps1

```diff
@@ -366,7 +366,7 @@ if ($includeCopy) {
                 "</result>"
     $Count = $failsSessionsBkC.Count
     Write-Output "<result>"
-                "  <channel>Failes-BackupCopys</channel>"
+                "  <channel>Failed-BackupCopys</channel>"
                 "  <value>$Count</value>"
```

Comment:
If the channel is renamed, Failes (Backup Job Error occurred but retry might happen) and Failed (completely failed) cannot be distinguished

Reply:
Hi - I have updated the code again as it was my bad for updating the channel name from Failes-BackupCopys to Failed-BackupCopys which is obviously a conflict with another channel name. I have updated that to Fails-BackupCopys

Reply:
> If the channel is renamed, Failes (Backup Copy Job Error occurred but retry might happen) and Failed (completely failed) cannot be distinguished

Hi @vMarkusK - have you had chance to review my latest commit and comments?